src/libs/actions/Policy.js

```diff
@@ -91,6 +91,54 @@ function getPolicyList() {
         });
 }
 
+/**
+ * Remove the passed members from the policy employeeList
+ *
+ * @param {Array} members
+ * @param {String} policyID
+ */
+function removeMembers(members, policyID) {
+
+    // Check if the members array is not empty
+    if (members.length === 0) {
+        return;
+    }
+
+    const key = `${ONYXKEYS.COLLECTION.POLICY}${policyID}`;
+
+    // Make a shallow copy to preserve original data and remove the members
+    const policy = _.clone(allPolicies[key]);
+    policy.employeeList = policy.employeeList.filter(email => !members.includes(email));
```

Comment:
Heads up, we default to using the `_` variety of array methods for consistency. 

In this case, however I'd recommend we use the [`_.without()` method](https://underscorejs.org/#without) instead of filter + includes.